packages/docs/docs/tutorials/bots/logistics-into-ehr.md

```diff
@@ -39,105 +38,78 @@ When we this implementation is complete, the whole workflow below will be done i
 
 * Make the Easypost subscription bot that will listen for webhooks
-  * First, [create a bot](https://app.medplum.com/Bot/new) called EasyPost Shipment Handler and save it
+  * First, [create a bot](https://app.medplum.com/admin/project) called EasyPost Shipment Handler and save it
   * Paste the code below into the Bot you created and save.
 
   ```js
-  //Grab the reference and the tracker object out of the Easypost Shipment json
-  const reference = input.get('reference');
-  const tracker = input.get('tracker');
-  const status = tracker.get('status');
-  const public_url = tracker.get('public_url');
-
-  //Find the ServiceRequest that has the Easypost reference
-  const serviceRequest = await medplum.readResource('ServiceRequest', reference);
-
-  if (!serviceRequest) {
-    const to_address = input.get('to_address');
-    const shipmentAddress = await medplum.createResource({
-      resourceType: 'Location',
-      identifier: [{
-        system: 'https://www.easypost.com/address',
-        value: to_address.get('id');
-      }],
-      name: to_address.get('name');
-      address: {
-        use: 'home',
-        type: 'physical',
-        line: [to_address.get('street1'), to_address.get('street2')],
-        city: to_address.get('city'),
-        state: to_address.get('state'),
-        postalCode: to_address.get('zip'),
-        country: to_address.get('country')
-      }
-    })
-    const newServiceRequest = await medplum.createResource({
-    resourceType: 'ServiceRequest',
-      identifier: [{
-        system: 'https://www.easypost.com/reference',
-        value: reference
-      }],
-      location: shipmentAddress, 
-      orderDetail: [
-        {"text": "easypost_tracker",
-        "coding": [
-          {
-            "system": "https://www.easypost.com/tracker",
-            "code": 'tracker',
-            "display": public_url
+  export async function handler(medplum, event) {
+    const input = event.input;
+    // The input is an Easypost Event https://www.easypost.com/docs/api#events
+    const reference = input['id'];
+    const status = input['status'];
+    const description = input['description'];
+    
+    // If this is not an Easypost tracker event, let's ignore it
+    if (description.toLowerCase().indexOf('tracker') === -1) {
+        console.log(reference + ' is not a tracking event');
+        return;
+    }
+    
+    const shipment_id = input['result']['shipment_id'];
+    const tracker = input['result']['tracking_code']; 
+    const public_url = input['result']['public_url'];
+    
+    //Find the ServiceRequest that has the Easypost reference
+    const serviceRequest = await medplum.searchOne('ServiceRequest?identifier=' + shipment_id);
+    
+    if (serviceRequest) {
```

Comment:
You can reduce the feeling of complexity if you invert this `if` statement and exit early.

For example:

```js
if (!serviceRequest) {
  console.log('Shipment ServiceRequest not found');
  return;
}
```